rocketpy/plots/rocket_plots.py

```diff
@@ -86,27 +86,31 @@ def stability_margin(self):
 
         return None
 
-    def power_on_drag(self):
-        """Plots power on drag of the rocket as a function of time.
-
-        Returns
-        -------
-        None
-        """
-
-        self.rocket.power_on_drag()
-
-        return None
-
-    def power_off_drag(self):
-        """Plots power off drag of the rocket as a function of time.
+    def power_on_and_off_drag(self):
```

Comment:
What do you think about `def drag_curves(self)` instead of `def power_on_and_power_off_drag`?

I think it's easier to do `my_rocket.plots.drag_curves()` than `my_rocket.plots.power_on_and_power_off_drag()`.
We keep the same meaning but in a more concise manner.

Reply:
I think it is a great suggestion ! I tried to be as clear as possible, but I forgot to be a little concise haha